### PR TITLE
Fix CompileLibraryResources cache miss.

### LINKED ISSUE #97: Fix Remote Cache miss with CompileLibraryResourcesTask
CompileLibraryResourcesTask.mergedLibraryResourcesDir uses absolute path as input.

https://issuetracker.google.com/issues/155218379

### PR DESCRIPTION
**Pull request by runningcode**
This is a regression from AGP 3.6.

Fixes #97

### CODE REVIEW

File: src/main/groovy/org/gradle/android/workarounds/CompileLibraryResourcesWorkaround.groovy [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    @Override
+    void apply(WorkaroundContext context) {
+        Project project = context.project
+        project.tasks.withType(androidTaskClass).configureEach { Task task ->
+            DirectoryProperty originalPropertyValue
+            // Create a synthetic input with the original property value and RELATIVE path sensitivity
+            project.gradle.taskGraph.beforeTask {
+                if (it == task) {
+                    println("configuring workaround before $task")
```
Comment (ghale) [inline src/main/groovy/org/gradle/android/workarounds/CompileLibraryResourcesWorkaround.groovy:39]: we need to remove this

File: build.gradle [hunk 2]
```diff
@@ -111,6 +111,7 @@ task install(type: Upload) {
 test {
     dependsOn install
     systemProperty "local.repo", localRepo.toURI()
+    maxParallelForks = 2
```
Comment (ghale) [inline build.gradle:114]: Did this actually improve anything for you?  I think I tried this out at one point and decided that with it running the android builds under the covers, it didn't actually speed things up (i.e. the bottleneck is IO and more parallelization in the tests didn't do anything to work around that).

File: src/main/groovy/org/gradle/android/workarounds/CompileLibraryResourcesWorkaround.groovy (hunk 1 above)
Comment (runningcode) [reply src/main/groovy/org/gradle/android/workarounds/CompileLibraryResourcesWorkaround.groovy:39]: haha good catch.

File: build.gradle (hunk 2 above)
Comment (runningcode) [reply build.gradle:114]: Yeah, you are right. There's almost no difference here in terms of performance of the tests. 
max parallel forks enabled:
https://scans.gradle.com/s/zmoo7agh4z23a/performance/build
max parallel forks disabled:
https://scans.gradle.com/s/ikgh3usjl3wwc/performance/build

Will remove.

**Pull request merged**